server/chat-plugins/scavengers.ts

```diff
@@ -362,6 +365,8 @@ export class ScavengerHunt extends Rooms.RoomGame<ScavengerHuntPlayer> {
 
 		this.hosts = hosts;
 
+		this.isHTML = isHTML ? isHTML : false;
```

Comment:
Any reason why this isn't just `this.isHTML = isHTML` or `this.isHTML = !!isHTML`?